react/components/TopMenu.js

```diff
@@ -253,56 +241,97 @@ class TopMenu extends Component {
     )
   }
 
+  renderCategoryMenu = (mobileMode) => (
+    <ExtensionPoint
+        id="category-menu"
+        mobileMode={mobileMode}
```

Comment:
`mobileMode`  shouldn't be passed to category-menu

Reply:
Fixed